### PR TITLE
Added filter to the pagination api

### PR DESCRIPTION
**Pull request by hero101**
### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/domain/community/user/user.service.ts [hunk 1]
```diff
@@ -519,6 +523,20 @@ export class UserService {
     return limitAndShuffle(users, limit, shuffle);
   }
 
+  async getPaginatedUsers(
+    paginationArgs: PaginationArgs,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (me-andre) [inline src/domain/community/user/user.service.ts]: are you sure you need this exception? `filter` seems to be used in the fn body.

**Review by me-andre — DISMISSED** [review]

**Comment by Comoque1** [comment]
Missing error messages for organizationPagination for preferences and groups
image

**Comment by Comoque1** [comment]
Add validation for invalid pagination values
image

**Review by valentinyanakiev — APPROVED** [review]
Looks good for building this further:
- rowID can be in the base alkemio interface (or alternatively in another base interface, but I think having SEED column everywhere is always beneficial).
- the SEED column rowID should be unsigned, and ideally UNSIGNED MEDIUMINT. It can't be negative. And int accommodates way more than needed anyway.
- how filtering is done should examined in the future (specific filters). Lookgs OK for now with LIKE.
- there are some functions (e.g. getRelayStylePaginationResults) that should be split as they are a bit too long.

Other than that, good approach and implementation :) Kudos.

**Comment by Comoque1** [comment]
Missing error message for usersPagination: `phone`
image

**Review by valentinyanakiev — CHANGES_REQUESTED** [review]
Check the [CI](https://app.travis-ci.com/github/alkem-io/server/builds/250037086), it seems you have missed an export

**Review by valentinyanakiev — APPROVED** [review]

**Pull request merged**